external/craigslist/tests.py

```diff
@@ -0,0 +1,19 @@
+from django.test import TestCase
+from .fetch import fetch_craigslist_housing
+from unittest import mock
+
+
+class CraigslistTests(TestCase):
+    @mock.patch("external.craigslist.fetch.CraigslistHousing")
```

Comment:
does the CraigslistHousing stub need to be specified here?

Reply:
Yes it's another form of mock.patch which I learned this time. By doing this, we get `MockCraigslistHousing` as a parameter of the test function.

Reply:
got it!